File: 2022-2023/sourcecode/lab/lab5/CarsyAppJava/app/src/main/java/pl/udu/uwr/pum/carsyappjava/util/FormatterUtil.java
package pl.udu.uwr.pum.carsyappjava.util;

import java.text.DecimalFormat;
import java.time.format.DateTimeFormatter;

public final class FormatterUtil {
    private FormatterUtil(){}

    public static DateTimeFormatter dateFormatter = DateTimeFormatter.ofPattern("yyyy MMM dd");

    public static DecimalFormat decimalFormat = new DecimalFormat("#.##");
}

File: 2022-2023/sourcecode/lab/lab5/MyFinanceAppKotlin/app/src/main/java/pl/edu/uwr/pum/myfinanceappkotlin/util/FormatterUtil.kt
package pl.edu.uwr.pum.myfinanceappkotlin.util

import java.text.DecimalFormat
import java.text.NumberFormat
import java.time.format.DateTimeFormatter

val formatter: NumberFormat = DecimalFormat("#,###.##")
val dateFormatter: DateTimeFormatter = DateTimeFormatter.ofPattern("yyyy MMM dd")
